.travis.yml

```diff
@@ -12,6 +12,10 @@ matrix:
           python: 3.7
           dist: xenial
           sudo: true
+        - os: linux
+          python: 3.8
+          dist: xenial
+          sudo: true
```

Comment:
The `sudo` keyword had been deprecated quite a while ago - it does nothing any more and should be removed.

Reply:
Yes, I've been removing both `dist: xenial` and `sudo: true` as I upgrade the travis ymls.